src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardProps {
+  header?: React.ReactNode
+  media?: React.ReactNode
+  footer?: React.ReactNode
+  standardLayout?: 'default' | 'headerFirst'
+  flagLayout?: 'default' | 'mediaOnRight'
+}
+
+export const Card = (
+  props: CardProps & React.HTMLAttributes<HTMLDivElement>
+): React.ReactElement => {
+  const {
+    header,
+    media,
+    footer,
+    standardLayout,
+    flagLayout,
+    children,
+    className,
+    ...divProps
+  } = props
+
+  // TODO determine how to implement the tablet:grid-col-4 class properly, should this be li class?
+  const classes = classnames(
+    'usa-card',
+    {
+      'usa-card--header-first': standardLayout == 'headerFirst',
+      'usa-card--flag': flagLayout != null,
+      'usa-card--media-right': flagLayout == 'mediaOnRight',
+    },
+    className
+  )
+
+  return (
+    <li className={classes} data-testid="Card">
+      <div className="usa-card__container" {...divProps}>
+        {header}
+        {media}
+        {children}
```

Comment:
The card body styles are important to make sure the flex layout works for this component overall. I think we should pull the styling into our Card component - something like this-   `<div className="usa-card__body">{children}</div>` instead.

Reply:
I think our header/footer styles could also be in the main component for the same reason of making the component act as a styles wrapper.  Something like  `header && <header className="whateverClasses>{header}</header>` in the case of the header.

Reply:
That makes sense :)

Reply:
I am going to keep the header, footer, and the body? as separate components so they are most customizable.

Reply:
I have made the header and footer as separate components. @haworku @suzubara Do you think there should be media and body components as well?

Reply:
@suzubara and I discussed the structure of the card component and came up with the following three possibilities:
```
<Card>   
    <CardHeader className="thing">...</CardHeader> 
</Card>  

<Card   header={<CardHeader className="thing">...</CardHeader>}   />

  <Card   header="MY header"   headerClasses="thing" />

```
We decided to go with the first option for simplicity. This will result in header, footer, media and body components.